lib/dynflow/connectors/direct.rb

```diff
@@ -3,6 +3,11 @@ module Dynflow
   module Connectors
     class Direct < Abstract
 
+      def prune_undeliverable_envelopes(_world)
```

Comment:
Could we move this one after the `Core` class definition with the other class methods?
Not sure if it's the correct place, but at least to be consistent :)